src/Makefile

```diff
@@ -64,6 +64,9 @@ FLAGS += -DUSE_POLLY
 ifeq ($(USE_POLLY_OPENMP),1)
 FLAGS += -fopenmp
 endif
+ifeq ($(USE_POLLY_ACC),1)
+LLVMLINK += -lPollyPPCG -Wl,-rpath=$(shell $(LLVM_CONFIG_HOST) --libdir),--no-as-needed -lGPURuntime -Wl,--as-needed
```

Comment:
> The GPURuntime headers to not require CUDA headers, so your patch should not need them either.
GPUJIT.c has a `#include <cuda.h>` and `#include <cuda_runtime.h>` and the path to them is included here `https://github.com/llvm-mirror/polly/blob/99a2e0c6bb2713e700c33cec48a9e8dd0461f9f6/CMakeLists.txt#L168`.

> I am also surprised you look at the source path of LLVM. I am not sure the source path will always be available. If we use the GPURuntime the headers should probably be installed and can then be used with the normal LLVM include paths, no?
I didn't get you here.